packages/electron-updater/src/AppUpdater.ts

```diff
@@ -390,6 +391,15 @@ export abstract class AppUpdater extends (EventEmitter as new () => TypedEmitter
       return false
     }
 
+    const minimumSystemVersion = updateInfo?.minimumSystemVersion
+    if (minimumSystemVersion) {
+      const currentOSVersion = release()
+      if (currentOSVersion && isVersionLessThan(currentOSVersion, minimumSystemVersion)) {
+        this._logger.info(`Current OS version ${currentOSVersion} is less than the minimum OS version required ${minimumSystemVersion} for version ${latestVersion}`)
+        return false
+      }
+    }
```

Comment:
I think this may need a try-catch in case an invalid `minimumSystemVersion` is provided

```
const minimumSystemVersion = updateInfo?.minimumSystemVersion
if (minimumSystemVersion) {
  try {
    const currentOSVersion = release()
    if (currentOSVersion && isVersionLessThan(currentOSVersion, minimumSystemVersion)) {
      this._logger.info(`Current OS version ${currentOSVersion} is less than the minimum OS version required ${minimumSystemVersion} for version ${latestVersion}`)
      return false
    }
  } catch (e: any) {
    // log and ignore
    this._logger.warn(`Unable to compare current OS version ${currentOSVersion} to minimum app version ${minimumSystemVersion}. ${ e.message })
  }
}
```

Reply:
Yes, we must add try-catch.